cmd/justification/main.go

```diff
@@ -63,13 +64,32 @@ func realMain(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("failed to setup kms client: %w", err)
 	}
+	defer func(kmsClient *kms.KeyManagementClient) {
+		err := kmsClient.Close()
+		if err != nil {
+			logger.Errorf("failed to close kms client with error %w", kmsClient)
+		}
+	}(kmsClient)
 
 	authHandler, err := grpcutil.NewJWTAuthenticationHandler(ctx, grpcutil.NoJWTAuthValidation())
 	if err != nil {
 		return fmt.Errorf("failed to setup grpc auth handler: %w", err)
 	}
 
-	p := justification.NewProcessor(kmsClient, cfg, authHandler)
+	firestoreClient, err := firestore.NewClient(ctx, cfg.FirestoreProjectID)
+	if err != nil {
+		return fmt.Errorf("failed to create Firestore client: %w", err)
+	}
+
+	defer func(fireStoreClient *firestore.Client) {
+		err := firestoreClient.Close()
+		if err != nil {
+			logger.Errorf("failed to close firestore client with error %w", kmsClient)
+		}
+	}(firestoreClient)
+
+	fireStoreRemoteConfig := config.NewFirestoreRemoteConfig(firestoreClient, "JVS/JustificationConfig")
```

Comment:
This is not the same as the justification_config, correct? Might be better named something like KeyConfig or something else.

Reply:
done